### PR TITLE
Added telesign SMS verify & Genius API (music)

### PR DESCRIPTION
**Pull request by AshishSharma1203**

### CODE REVIEW

File: src/content/apis/sports/API-FOOTBALL.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
```
Comment (TheLearneer) [inline src/content/apis/sports/API-FOOTBALL.md:1]: This API already exists under `sports API`

Comment (TheLearneer) [reply src/content/apis/sports/API-FOOTBALL.md:1]: Make sure to check the website and you will see it there 🙏 
Screen Shot 2022-10-25 at 9 20 53 PM

Comment (AshishSharma1203) [reply src/content/apis/sports/API-FOOTBALL.md:1]: Pardon my mistake 😶

**Review by TheLearneer — CHANGES_REQUESTED** [review]
Make sure to comply with requirements as mentioned in #2  if you are here for hacktoberfest 🙏

**Comment by AshishSharma1203** [comment]
> Make sure to comply with requirements as mentioned in #2 if you are here for hacktoberfest 🙏

Can you check it now .

File: src/content/apis/SMS/telesign SMS Verify .md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
```
Comment (TheLearneer) [inline src/content/apis/SMS/telesign SMS Verify .md:1]: Can you please update the file names as per the other files that are available.
And also please upload the file under `miscs` directory

File: src/content/apis/SMS/telesign SMS Verify .md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+name: Telesign SMS Verify API
+description: This API Delivers patented phone-based verification and two-factor authentication using a time-based, one-time passcode sent over SMS
+categories:
+  - SMS
```
Comment (TheLearneer) [inline src/content/apis/SMS/telesign SMS Verify .md:5]:
```suggestion
  - miscs
```

File: src/content/apis/music/Genius-Song Lyrics.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+---
```
Comment (TheLearneer) [inline src/content/apis/music/Genius-Song Lyrics.md:1]: Please properly format the file name like other files 🙏

**Review by TheLearneer — CHANGES_REQUESTED** [review]

File: src/content/apis/miscs/telesign-sms-verify.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+---
+name: Telesign SMS Verify API
+description: This API Delivers patented phone-based verification and two-factor authentication using a time-based, one-time passcode sent over SMS
+categories:
+  - misc
```
Comment (TheLearneer) [inline src/content/apis/miscs/telesign-sms-verify.md:5]:
```suggestion
  - miscs
```

Comment (AshishSharma1203) [reply src/content/apis/miscs/telesign-sms-verify.md]: done

**Review by TheLearneer — APPROVED** [review]
Thank you for your contribution

**Pull request merged**